db/data/20230831064255_add_facility_business_identifiers_in_ethiopia.rb

```diff
@@ -0,0 +1,102 @@
+    "1c0a392d-a298-42ae-9731-4bf47f40f12d" => "sbHC7FgFWLb",
+    "e852a982-a813-44a1-b9dd-1219ed8bcdb9" => "RuhWkurzxXy",
+    "d0f1ed68-20d5-4dae-9a52-f5faf7a6def4" => "XidDZoPPqj6",
+    "8306b482-8bbe-46a7-becb-1d47db7bbc06" => "XGRbsdt9xsE",
+    "2b55c98d-1380-4a23-8ac7-bbe7c16d3544" => "C8HFOWHlarl",
+    "413e915f-47e6-4379-a5fc-be3fad393e06" => "htGs3ST4c55",
+    "3e22120c-360b-4ad6-82f0-2960349c62e8" => "ut2BTKG9j7E",
+    "742ace6e-e86d-4a78-b1df-c9d4f0c21e9c" => "KuTv8c1gqLv",
+    "28be1b5f-9a77-4e85-800d-7a1f5f061b11" => "vdWDpmjXiod",
+    "2b70df68-4cfe-4767-9234-cc85fa13a927" => "uUFu8MmC1N2",
+    "1b5ed107-8bfd-42b6-bc23-2664f6f781ab" => "ijWbyhFFCx3",
+    "2bed45c4-8dc7-40f8-8b49-3653a5cbab6b" => "mXy5MKakCXX",
+    "b9a30acc-a243-46a4-accc-afeeb4b96784" => "O1gFRqSqBpY",
+    "d1d0be81-3b9c-42e5-ac40-876153e8c74e" => "MEJYJtqSqlT",
+    "cd1f179b-8276-45a8-9fac-1bc9dd71ba30" => "SOw1j01Dk2d",
+    "e10d28b2-204f-40e8-b252-3cce8444f558" => "hASDJs1l7Uc",
+    "ab10cef1-1742-4305-86e5-5e3a904bd62b" => "MUnfwQryhZ5",
+    "e5a9bdc1-7302-4e34-b067-283ebde08469" => "SK7FGJGfOt2",
+    "67bbe898-5224-4b46-adeb-dddf72456808" => "qn8ptV5Prqb",
+    "b8337ca6-5f1a-4528-9845-def772e84a7c" => "SDyh2spIBU3",
+    "f7e4cb80-bd35-4248-87fb-532c88a69468" => "eNwpyeo0BOT",
+    "43db12e2-e14c-446f-b5f4-be6ef4ddf951" => "Vcof25JT4PJ",
+    "0c015f80-bf09-4b21-98f4-274990e21a86" => "BKVCeW4q361",
+    "2d1ea6f4-37bb-4850-a5db-b0bada9781fc" => "DKsubMyV5Nd",
+    "cab61ca6-50c3-4a49-9071-ca21d7a56c7c" => "EhFRIiSwwgx",
+    "d2ed8ddd-8f4c-4dd1-8f7b-4e9151daba81" => "EjT3LKUyale",
+    "f8284384-7d27-4f03-b855-c5eac4116454" => "g3Sz2VRqAo7",
+    "28208165-f462-4602-ad63-c39d59979556" => "QAVjKCqVAXA",
+    "e8a4d9c8-5ec0-498f-a5b2-353550f895c4" => "mez1nztRYbc",
+    "28582cac-da41-4ae8-9cf7-73577b0ac1df" => "n9oGQflGqnT",
+    "209e38d3-8cd4-4b03-98bf-63589895c699" => "d7xLvXrmM24"
+  }
+
+  def up
+    unless CountryConfig.current_country?("Ethiopia") && ENV["SIMPLE_SERVER_ENV"] == "production"
+      print "This migrations is meant to run only in Ethiopia production"
+      return
+    end
+
+    FACILITY_IDENTIFIERS.each do |simple_id, dhis2_org_id|
+      facility = Facility.find_by(id: simple_id)
+      unless facility.present?
+        print "Facility not found: id #{simple_id}"
+        next
+      end
+
+      facility.business_identifiers.find_or_create_by(
+        identifier_type: :dhis2_org_unit_id,
+        identifier: dhis2_org_id
+      )
+    end
+  end
+
+  def down
+    unless CountryConfig.current_country?("Ethiopia") && ENV["SIMPLE_SERVER_ENV"] == "production"
+      print "This migrations is meant to run only in Ethiopia production"
+      return
+    end
+
+    FACILITY_IDENTIFIERS.each do |simple_id, dhis2_org_id|
+      facility = Facility.find_by(id: simple_id)
+      unless facility.present?
+        print "Facility not found: id #{simple_id}"
+        next
+      end
+
+      facility.business_identifiers.find_by(
```

Comment:
This down command will delete records which were not created by this migration because of the `find_or_create_by` command. We can include a `created_at` > 4 Sep to avoid that, maybe.

Reply:
I think it's fine since we are specifically deleting the records we are creating using both simple and dhis2 ids.